app.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const app = express()
+const { getTopics } = require("./controllers/topics.controllers")
+
+app.use(express.json())
+
+
+app.get('/api/topics', getTopics);
+
+app.use((err, request, response, next) => {
+    if(err.status && err.msg) {
+        response.status(err.status).send({msg: err.msg})
+    }
+    next(err)
+})
```

Comment:
Whilst the 500 catch all below is boilerplate code and can be written straight away. The error handling above will only be necessary in a couple of tickets time and probably shouldn't be here yet